extensions/wikia/WikiaPhotoGallery/WikiaPhotoGalleryHelper.class.php

```diff
@@ -94,8 +94,8 @@ static public function setupEditPage($editform) {
 
 		wfProfileIn( __METHOD__ );
 
-		if ( F::app()->checkSkin( [ 'oasis' ] ) ) {
-			$app->wg->Out->addScript("<script type=\"{$app->wg->JsMimeType}\" src=\"{$app->wg->ExtensionsPath}/wikia/WikiaPhotoGallery/js/WikiaPhotoGallery.js\"></script>\n");
+		if ( $app->checkSkin( [ 'oasis' ] ) ) {
+			$app->wg->Out->addModules( 'ext.WikiaPhotoGallery' );
 
 			// load message for MW toolbar button tooltip
 			$wgHooks['MakeGlobalVariablesScript'][] = 'WikiaPhotoGalleryHelper::makeGlobalVariablesScript';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgHooks" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)